packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1856,7 +1803,6 @@ module.exports = function (log, config, bounces) {
         productPaymentCycleNew,
         productPaymentCycleOld,
         icon: productIconURLNew,
-        product: productNameNew,
```

Comment:
We don't reference `product` in any templates so I removed them here.

`translatorParams` is no longer needed since we have `flattenNestedObjects` now.

`productNameNew` was changed to `productName` for consistency - we were previously checking for (psuedo code) `if (!productName && productNameNew) { productName = productNameNew }` in our templates so we don't need that anymore now.

There's also at least one instance where I removed passing in `subscription` since we don't reference it and only need `productName`. I'm actually not sure if we ever even need to pass in the entire `subscription` object since I don't see that we ever need `planId` or `productId` in the templates.